x-pack/platform/plugins/shared/lens/public/visualizations/xy/xy_config_panel/index.tsx

```diff
@@ -11,10 +11,14 @@ import { LegendValue, Position, ScaleType } from '@elastic/charts';
 import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import { AxisExtentConfig, YScaleType } from '@kbn/expression-xy-plugin/common';
 import { TooltipWrapper } from '@kbn/visualization-utils';
-import { LegendSize, XYLegendValue } from '@kbn/visualizations-plugin/common/constants';
+import {
+  LegendSize,
+  type XYLegendValue,
+  type AxesSettingsConfig,
+} from '@kbn/visualizations-plugin/common';
```

Comment:
```suggestion
import { LegendSize } from '@kbn/visualizations-plugin/common/constants';
import type { AxesSettingsConfig, XYLegendValue } from '@kbn/visualizations-plugin/common';
```
this line is causing the bundling error, since `common/` folder is not public (you cannot import modules from the outside to this folder). 
<img width="1370" alt="Screenshot 2025-02-06 at 13 37 30" src="https://github.com/user-attachments/assets/13e5eb47-0afe-41b9-8b2b-f3ae55b48765" />

There are two ways to fix it - either import the types directly from the folder that is allowed to do so (check out `src/platform/plugins/shared/visualizations/kibana.jsonc` to know which ones are those, essentially:

```
  "extraPublicDirs": [
      "common/constants",
      "common/utils",
      "common/expression_functions",
      "common/convert_to_lens"
    ]
```

or add this file to it (common/index). I wouldn't recommend the second because it can increase the bundle.

Reply:
Done here: [Fix import](https://github.com/elastic/kibana/pull/208002/commits/db9a9a9421ea3a68e478850be6b9b1ebc4c34233). Thanks for raising this!